### DISCUSSION TITLE
Migrate from `GaugeBuilder` to `GaugeGenerator`

### DISCUSSION THREAD

**Discussion by MichaelSL** [Live-Charts/LiveCharts2#1185] [Q&A]
I've tried to update LiveChart2 for MAUI for the latest and found that `GaugeBuilder` is not supported anymore (it still works on Android but doesn't render anything on iOS).

So I've decided to upgrade to `GaugeGenerator`. I'm trying to replicate my previous setup and it seems like not all properties are avalable:

```csharp
speedControl._series = new GaugeBuilder()
    .WithLabelsSize(25)
    .WithInnerRadius(70)
    .WithBackgroundInnerRadius(70)
    .WithLabelFormatter(x => $"{x.Coordinate.PrimaryValue} {(AppPreferences.IsImperialUnits ? "mph" : "km/h")}")
    .AddValue(newValDouble)
    .BuildSeries();

var k = GaugeGenerator.BuildSolidGauge(new GaugeItem(newValDouble, seriesBuilder =>
{
    seriesBuilder.DataLabelsSize = 25;
    seriesBuilder.InnerRadius = 70;
    // seriesBuilder.BackgroundInnerRadius = 70;        not avalable
    // seriesBuilder.LabelFormatter = x => $"{x.Coordinate.PrimaryValue} {(AppPreferences.IsImperialUnits ? "mph" : "km/h")}";        not available
}));
```

Does anyone already migrated to the new generator? Please help me to replicate my previous setup.

**Comment by beto-rodriguez** [comment]
Hello, it works on iOS, maybe you are facing #800?

**Reply by MichaelSL** [reply]
nope, it doesn't look like it. I deploy from the Mac and it worked fine before the update.
I've reverted to the previous version and deployed successfully without disconnecting the phone from a Mac.
`<PackageReference Include="LiveChartsCore.SkiaSharpView.Maui" Version="2.0.0-beta.850.1" />` - version that works fine

**Reply by beto-rodriguez** [reply]
I will take a look into that, now docs in the web site are updated to use the GaugeGenerator.
